pom.xml

```diff
@@ -41,35 +41,33 @@
 
         <product.build-time>${timestamp}</product.build-time>
 
-        <wildfly.version>18.0.1.Final</wildfly.version>
-        <wildfly.build-tools.version>1.2.12.Final</wildfly.build-tools.version>
-        <eap.version>7.3.0.GA-redhat-00004</eap.version>
-        <wildfly.core.version>10.0.3.Final</wildfly.core.version>
+        <wildfly.version>19.0.0.Final</wildfly.version>
+        <wildfly.build-tools.version>1.2.13.Final</wildfly.build-tools.version>
+        <eap.version>7.4.0.CD19-redhat-00001</eap.version>
```

Comment:
Please keep old version 7.3.0.GA-redhat-00004 for now. Build with new one is broken in PNC. Sorry for confusion.

Reply:
Ok, reverted